website/content/en/docs/dev/Internals/_index.md

```diff
@@ -37,6 +37,7 @@ Metadata:
 - `protected`: empty file, used by `limactl protect`
 
 cloud-init:
+- `cloud-config.yaml`: cloud-init configuration.
```

Comment:
```suggestion
- `cloud-config.yaml`: cloud-init configuration. Modification to this file will be just discarded.
```

Reply:
Made the file readonly (444) and added a comment